tests/utils.py

```diff
@@ -27,14 +27,14 @@ def assert_not_equal(x, y):
     assert not (x == y)
 
 
-def read_crypto_test_vectors(fname, maxels=0):
+def read_crypto_test_vectors(fname, maxels=0, delimiter=b'\t'):
```

Comment:
Sorry one more comment: could we just update the call sites so we don't need a default value for delimiter here?

Reply:
I agree with updating callers, but I'd rather keep delimiter as a keyword argument. Update coming.

Reply:
Since #317 adds another caller, the one getting last must be rebased on the other.

Reply:
Let's land this refactor PR with the update and the other one can be rebased.

Reply:
Just pushed the update.